src/sidebar/templates/annotation-thread.html

```diff
@@ -14,25 +14,13 @@
       annotation="vm.thread.annotation"
       ng-if="vm.thread.annotation">
     </moderation-banner>
-    <annotation-omega ng-if="vm.shouldShowAnnotationOmega() && vm.thread.annotation"
+    <annotation-omega ng-if="vm.thread.annotation && vm.thread.visible"
```

Comment:
Note the conditional here for the `visible` change mentioned in a previous comment.